modules/policy/basic-policies/src/test/resources/features/SWHSourceIdKnownRule.feature

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+Feature: Software Heritage link available
+  Artifacts should have a link to the sources on software heritage.
+
+    Scenario: Artifact without Software Heritage information fails the rule
+      Given an artifact
+      When I use the rule "SWHAvailable"
+      Then the artifact should fail with policy id "SWHAvailable"
+
+    Scenario: Proprietary artifact passes the rule
+      Given an artifact with
+        | proprietary | true |
+      When I use the rule "SWHAvailable"
+      Then no artifact fails on policy id "SWHAvailable"
+
+    Scenario: Artifact with Software Heritage information pass the rule
+      Given an artifact with
+        | sources | swh:1:rel:22ece559cc7cc2364edc5e5593d63ae8bd229f9f |
```

Comment:
shouldn't sources be a URL/URI?

Reply:
It is only a vehicle to create an artifact with the right properties. SWH-Ids are used for storage of SWH information, so if you want to set sources as a SWH information, I thought this is the right format.

Reply:
How does this Code, which uses `| sources | ` set something of the type `ArtifactSoftwareHeritageID` in the artifact? As far as I understand https://github.com/eclipse/antenna/pull/313/files/17fb26f37a07aed80e777bb4c28ba4ecbf5111ad#diff-83c00c25c943424b203013f5641ed1b7R117-R120 this would be necessary.

Reply:
Ok, I have found https://github.com/eclipse/antenna/pull/313/files/17fb26f37a07aed80e777bb4c28ba4ecbf5111ad#diff-83c3878db98c05b17f595be186bc0edaR171-R174. An url would be https://archive.softwareheritage.org/swh:1:rel:22ece559cc7cc2364edc5e5593d63ae8bd229f9f/.

Why not use another mechanism as a setter, instead of `sources` with magic. Then it would be easier to understand.